frontend_app_folder_advanced/src/container/FolderAdvanced.jsx

```diff
@@ -295,13 +299,25 @@ export class FolderAdvanced extends React.Component {
 
         <PopinFixedOption>
           <div className='folder_advanced__header'>
-            {state.showRefreshWarning && (
-              <RefreshWarningMessage
-                tooltip={props.t('The content has been modified by {{author}}', { author: state.editionAuthor, interpolation: { escapeValue: false } })}
-                onClickRefresh={this.handleClickRefresh}
+            <ToolBar>
+              <FavoriteButton
+                favoriteState={props.isContentInFavoriteList(state.content, state)
+                  ? FAVORITE_STATE.FAVORITE
+                  : FAVORITE_STATE.NOT_FAVORITE}
```

Comment:
```suggestion
                favoriteState={(
                  props.isContentInFavoriteList(state.content, state)
                    ? FAVORITE_STATE.FAVORITE
                    : FAVORITE_STATE.NOT_FAVORITE
                )}
```
This is only a suggestion. You might also want to put this in a constant. Or do nothing. I'm okay with the three outcomes.